packages/core/notarization/commitments.go

```diff
@@ -105,27 +104,17 @@ func (f *EpochCommitmentFactory) ECR(ei epoch.Index) (ecr epoch.ECR, err error)
 	branch1 := make([]byte, 0)
 	branch1a := make([]byte, 0)
 	branch1b := make([]byte, 0)
```

Comment:
You don't need these anymore. byteutils.ConcatBytes will allocate a new buffer for you.